pkgs/top-level/all-packages.nix

```
@@ -18447,6 +18447,8 @@ with pkgs;
 
   proj = callPackage ../development/libraries/proj { };
 
+  proj7 = callPackage ../development/libraries/proj/7.nix { };
```

Should this be called
```suggestion
  proj_7 = callPackage ../development/libraries/proj/7.nix { };
```
and should I use
```suggestion
  proj7 = proj.overrideAttrs (...)
```
instead?

---

I think it should be called ``proj_7`` and it is fine to use the absolute path. This should also e more performant.